static/styles/zulip.css

```diff
@@ -2831,12 +2853,34 @@ ul.expanded_subjects {
     padding-bottom: 2px;
 }
 
+ul.expanded_private_messages {
+    list-style-type: none;
+    font-weight: 300;
+    font-size: 84%;
+    margin-left: 0px;
+    padding-bottom: 2px;
+}
+
 li.show-more-topics,
 li.expanded_subject {
     position: relative;
     padding-left: 33px;
 }
 
+
+li.show-more-private-messages,
+li.expanded_private_message {
+    position: relative;
+    padding-left: 24px;
+    padding-bottom: 1px;
+    padding-top: 2px;
+}
+
+
+#global_filters li:hover {
+    background-color: #e2e8dd;
+}
```

Comment:
I confess to not noticing that I had added that - I thought I had accidentally removed it from the CSS in an editing glitch and didn't cross-check it with production. It was copied from the streams list to match its highlighting behaviour.